README.md

```diff
@@ -1,2 +1,196 @@
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [Live Demo](#Live-Demo) 
+  - [Video Walkthrogh](#Video-Walkthrogh) 
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 Capstone Project - MyBudget <a name="about-project"></a>
+
+- **Write integration tests with Capybara gem.**
+- **Utilize Ruby gems as a package management system.**
+- **Install the Ruby on Rails framework.**
+- **Grasp the principles of Rails' RESTful design and routing.**
+- **Employ controllers to handle HTTP requests and render views.**
+- **Safely handle and process parameters from browser requests.**
+- **Create views using preprocessed HTML files with embedded Ruby code.**
+- **Utilize layouts and templates for shared content and consistent design.**
+- **Manage database schema through migration files.**
+- **Implement data validation for models to maintain data integrity.**
+- **Mitigate performance issues, such as the N+1 query problem.**
+- **Understand the concept of Object-Relational Mapping (ORM).**
+- **Write SQL queries using ActiveRecord for database operations.**
+- **Establish associations between different data models.**
+- **Develop a web application with user authentication requirements.**
+- **Implement authentication using the Devise gem.**
+- **Control access to web application resources based on authorization rules.**
+- **Document and explain coding decisions for architectural choices.**
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+The various stacks used for the project includes the following
+<details>
+  <summary>Ruby</summary>
+  <summary>Ruby on Rails</summary>
+</details>
+
+
+
+### Key Features <a name="key-features"></a>
+
+
+- **Use Ruby Gems as a software packages system.**
+- **Understand what ORM is.**
+- **Use database migration files to maintain database schema.**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To clone and run this application, you'll need [Git](https://git-scm.com) and [Ruby](https://rubyinstaller.org/downloads/) installed on your computer.
+
+### Prerequisites
+From your command line:
+
+On macOS, by using [Homebrew](https://brew.sh/) you can install.
+
+```bash
+# install rbenv
+$ brew install rbenv
+```
+
+```bash
+# install ruby-build
+$ brew install ruby-build
+```
+
+On Windows, by using [chocolatey](https://chocolatey.org) you can install.
+
+```bash
+# install ruby
+C:\> choco install ruby
+```
+
+### Live Demo 
+<a href="https://mybudget-lf89.onrender.com/">Live Demo Link</a>
```

Comment:
- [ ] Good job with the deployment but when I use the deploy link and create a new account all data "categories and transactions" that have been created in another account are loaded in the new account. Normally when a user creates a new account the category page is supposed to be empty as it is a new account. Kindly fix that 🙏🏾 

<img width="371" alt="Screenshot 2023-10-21 at 18 28 12" src="https://github.com/yemidada/MyBudget/assets/74501523/586385dd-d4af-497f-a540-ccd6ad59115b">

- [ ] Also, when I was using your application, I had a problem, your application wasn't working properly and was crashing, please make sure your deployed link is working properly without any problems 🙃 

<img width="371" alt="Screenshot 2023-10-21 at 18 29 34" src="https://github.com/yemidada/MyBudget/assets/74501523/f7a03935-8874-4afe-95d4-180e9466d45d">